app/assets/javascripts/admin/addon/templates/components/ip-lookup.hbs

```diff
@@ -20,7 +20,11 @@
 
         <dt>{{i18n "ip_lookup.location"}}</dt>
         <dd>
-          {{#if this.location.location}}
+          {{#if this.location.no_license}}
+            <a href="https://meta.discourse.org/t/configure-maxmind-for-reverse-ip-lookups/173941" rel="noopener noreferrer" target="_blank">
+              {{i18n "ip_lookup.no_license"}}
```

Comment:
Where do I need to define the initial/English string for this?

Reply:
`/config/locales/client.en.yml`.

Reply:
Many thanks, I added a meaningful string, hope it's fine.